src/progress/nested/summary.rs

```diff
@@ -63,9 +63,9 @@ where TOuter: Timestamp,
 impl<SOuter: Display, SInner: Display> Display for Summary<SOuter, SInner> {
     #[inline]
     fn fmt(&self, f: &mut Formatter) -> Result<(), Error> {
-        match self {
-            &Local(ref s) => f.write_str(&format!("Local({})", s)),
-            &Outer(ref s, ref t) => f.write_str(&format!("Outer({}, {})", s, t))
+        match *self {
+            Local(ref s) => f.write_str(&format!("Local({})", s)),
+            Outer(ref s, ref t) => f.write_str(&format!("Outer({}, {})", s, t))
```

Comment:
I think they can even be calls to `write!(f, /* ... */)` (because `f` is already a mutable reference).